packages/api-derive/src/treasury/proposals.ts

```diff
@@ -1,14 +1,13 @@
 // Copyright 2017-2020 @polkadot/api-derive authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
-import type { Observable } from 'rxjs';
 import type { ApiInterfaceRx } from '@polkadot/api/types';
 import type { Option } from '@polkadot/types';
 import type { ProposalIndex, TreasuryProposal } from '@polkadot/types/interfaces';
-import type { DeriveCollectiveProposal, DeriveTreasuryProposal, DeriveTreasuryProposals } from '../types';
-
+import type { Observable } from 'rxjs';
```

Comment:
Side-comment, all these are weird - this is where I actually dislike fix, since fix will move to the correct places, pass the linting and hurt my eyes :)